src/boomer.nim

```diff
@@ -110,7 +110,14 @@ proc main() =
     CWColormap or CWEventMask, addr swa)
 
   discard XMapWindow(display, win)
-  discard XStoreName(display, win, "Wordpress Application")
+
+  var wmName = "boomer"
+  var wmClass = "Boomer"
+  var hints = TXClassHint(res_name: wmName, res_class: wmClass)
+  var pHints: PXClassHint = addr(hints)
```

Comment:
unless you're actively reusing the pointer, its more idiomatic to inline this.

Reply:
I know, i am sorry, but and as i said, this is my second time trying NIM (probably in total less than 30 minutes exposure). I just needed WM_CLASS and i saw i can make it, and i made is like this. Sorry again.

Reply:
brother, im just giving a hint, no need to be sorry.

Reply:
Thank you. :)
You are definitely right actually. And I actually tried to go through the NIM docs (with very little time to spare) and try and do that, but coudn't find. Would you mind sharing, how can one do just that?
Thanks again for the hint :)

Reply:
i guess my wording wasn't most precise. i meant removing the pHints declaration and passing `addr hints` to `XSetClassHint` directly.